src/web/lib/components/SavedThemeSelector/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import BrowserPreview from '../BrowserPreview';
+
+import './index.scss';
+
+export const SavedThemeSelector = ({ setTheme, savedThemes, deleteTheme }) => {
+  const sortedSavedThemes = Object.entries(savedThemes).sort(
```

Comment:
Could move this to a selector in store.js, but punted for now to make it easier to play with the presentation

Reply:
Fair.